src/main/java/org/terasology/launcher/LauncherInitTask.java

```diff
@@ -285,20 +275,6 @@ private Path getGameDataDirectory(OperatingSystem os, Path settingsGameDataDirec
         return gameDataDirectory;
     }
 
-    private TerasologyGameVersions getTerasologyGameVersions(Path launcherDirectory, Path gameDirectory, BaseLauncherSettings launcherSettings) {
```

Comment:
Not sure whether this should be removed already (@praj-foss ?). I just threw it out because it was not used... Can re-add it easily from Git history if needed, though.